doc/source/tutorial/optimize.rst

```diff
@@ -522,26 +522,26 @@ The nonlinear constraint:
    :nowrap:
 
      \begin{equation*} c(x) =
-     \left[\begin{array}{c} x_0^2 + x_1 \\ x_0^2 - x_1\end{array}\right]
+     \begin{bmatrix} x_0^2 + x_1 \\ x_0^2 - x_1\end{bmatrix}
       \leq 
-      \left[\begin{array}{c} 1 \\ 1\end{array}\right], \end{equation*}
+      \begin{bmatrix} 1 \\ 1\end{bmatrix}, \end{equation*}
 
 with Jacobian matrix:
 
 .. math::
    :nowrap:
 
      \begin{equation*} J(x) =
-     \left[\begin{array}{c} 2x_0 & 1 \\ 2x_0 & -1\end{array}\right],\end{equation*}
+     \begin{bmatrix} 2x_0 & 1 \\ 2x_0 & -1\end{bmatrix},\end{equation*}
 
 and linear combination of the Hessians:
 
 .. math::
    :nowrap:
 
      \begin{equation*} H(x, v) = \sum_{i=0}^1 v_i \nabla^2 c_i(x) =
-     v_0\left[\begin{array}{c} 2 & 0 \\ 0 & 0\end{array}\right] + 
-     v_1\left[\begin{array}{c} 2 & 0 \\ 0 & 0\end{array}\right],
```

Comment:
This is mark-up bug, should have been ``{cc}`` and in two previous lines too. But better overall to use directly ``bmatrix`` environment which is well rendered by MathJax and understood by Sphinx for LaTeX, so I changed all instances in the file to use ``bmatrix``, not only those faulty lines.